### PR TITLE
[GNA] [TLV] [CVF] Enabling model export

### PR DESCRIPTION
**Pull request by kbruniec**
### Details:
 - TLV support for GNA 3.1

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/tests/functional/shared_test_classes/src/subgraph/split_conv_concat.cpp [hunk 1]
```diff
@@ -48,4 +48,80 @@ void SplitConvConcat::SetUp() {
...
+                                                  {kx, ky},
+                                                  {1, 1},
+                                                  {0, 0},
+                                                  {0, 0},
+                                                  {1, 1},
+                                                  ngraph::op::PadType::VALID,
+                                                  outCh,
+                                                  false,
+                                                  filterWeights1);
+    auto relu1 = std::make_shared<ngraph::opset1::Relu>(conv1);
```
Comment (nkogteva) [inline src/tests/functional/shared_test_classes/src/subgraph/split_conv_concat.cpp:94]: Use latest opset

File: src/tests/unit/gna/gna_memory_compact_test.cpp [hunk 2]
```diff
@@ -12,13 +12,15 @@ using namespace GNAPluginNS::memory;
...
  protected:
-    GNAMemory<std::allocator<uint8_t>> mem;
+    GNAMemory<GNAPluginNS::memory::GNAFloatAllocator> mem;
     bool isCompact = true;
 
     void SetUp() override  {
     }
 };
 
+/* Enable compact mode tests
```
Comment (nkogteva) [inline src/tests/unit/gna/gna_memory_compact_test.cpp:22]: Who is working on it? @mryzhov

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**